### PR TITLE
Reinstate PDF validation for Claims endpoints

### PR DESCRIPTION
**Pull request by hinzed1127**
## Description of change

This adds PDF validation back into claims. Previously it was only getting run for document validation. This also adds it to the `upload_526` method, as well.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

### CODE REVIEW

File: modules/claims_api/app/controllers/concerns/claims_api/document_validations.rb [hunk 1]
```diff
@@ -32,10 +32,12 @@ def document_content_type_errors
         end
       end
 
+      # rubocop:disable Layout/LineLength
       def pdf?(document)
         extension = document.original_filename.split('.').last
-        ['application/pdf', 'text/plain'].include?(document.content_type) && extension.downcase == 'pdf'
+        ['application/pdf', 'text/plain', 'application/octet-stream'].include?(document.content_type) && extension.downcase == 'pdf'
```
Comment (hinzed1127) [inline modules/claims_api/app/controllers/concerns/claims_api/document_validations.rb:38]: reason for this addition is a ruby quirk. basically if `Content-Type` is unspecified, it defaults to `application/octet-stream`. I think if the extension is still `pdf` this is a safe thing to allow

File: modules/claims_api/app/controllers/concerns/claims_api/document_validations.rb [hunk 2]
```diff
@@ -32,10 +32,12 @@ def document_content_type_errors
         end
       end
 
+      # rubocop:disable Layout/LineLength
```
Comment (SlakrHakr) [inline modules/claims_api/app/controllers/concerns/claims_api/document_validations.rb:35]: can we not just bump the second clause down to the next line to get around the LineLength cop?

**Review by SlakrHakr — APPROVED** [review]

File: modules/claims_api/app/controllers/concerns/claims_api/document_validations.rb (hunk 2 above)
Comment (hinzed1127) [reply modules/claims_api/app/controllers/concerns/claims_api/document_validations.rb:35]: good point, I'll make that update before merging

**Review by bastosmichael — APPROVED** [review]
Good job checking each other's PR's gents.

**Pull request merged**